source/lib/cdk-textract-stack.ts

```diff
@@ -258,31 +260,31 @@ export class CdkTextractStack extends cdk.Stack {
             dedicatedMasterEnabled: true,
             zoneAwarenessEnabled: true,
             zoneAwarenessConfig: {
-              availabilityZoneCount: 2
-            }
+              availabilityZoneCount: 2,
+            },
           },
           ebsOptions: {
             ebsEnabled: true,
             volumeSize: 20,
-            volumeType: "gp2"
+            volumeType: "gp2",
           },
           encryptionAtRestOptions: {
             enabled: true,
-            kmsKeyId: esEncryptionKey.keyId
+            kmsKeyId: esEncryptionKey.keyId,
           },
           nodeToNodeEncryptionOptions: {
-            enabled: true
+            enabled: true,
           },
           logPublishingOptions: {
             INDEX_SLOW_LOGS: {
               cloudWatchLogsLogGroupArn: esLogGroup.logGroupArn,
```

Comment:
Where do we set the permission for the ES Service Role?
We need to provide permissions to ES Service Role to write to the esLogGroup 
https://aws.amazon.com/blogs/big-data/viewing-amazon-elasticsearch-service-error-logs/

Reply:
This is out of scope for this PR. I merely added node to node encryption to render security the same on both deployment types.